### PR TITLE
Generate SQL for insertRecordMany

### PR DESCRIPTION
**Pull request by cmika**
Due to how `executeMany` is implmented the result of calling
`insertRecordMany` would be multiple calls to the database. This reworks
`insertRecordMany` so instead it generates a multi record insert
statement and only makes 1 call.

### CODE REVIEW

File: orville-postgresql/src/Database/Orville/PostgreSQL/Internal/Sql.hs [hunk 1]
```diff
@@ -9,11 +9,23 @@ import qualified Data.List as List
 
 mkInsertClause :: String -> [String] -> String
 mkInsertClause tblName columnNames =
```
Comment (qxjit) [inline orville-postgresql/src/Database/Orville/PostgreSQL/Internal/Sql.hs:11]: Could `mkInsertClause` now just call `mkInsertManyClause` with a `recordCount` of 1?

**Review by qxjit — APPROVED** [review]
It might be nice to have a direct test of somewhere rather than relying on the usage in the popper tests.

File: orville-postgresql/src/Database/Orville/PostgreSQL/Internal/Sql.hs (hunk 1 above)
Comment (cmika) [reply orville-postgresql/src/Database/Orville/PostgreSQL/Internal/Sql.hs:11]: I thought about that and forgot to change it lol

**Review by exodrifter — COMMENTED** [review]
Do you also intend to fix this in [`Database.Orville.Oracle.Core.insertRecordMany`](https://github.com/flipstone/orville/blob/431f9aecca49dfcedb47c1d65a0c9bbc79faa191/orville-oracle/src/Database/Orville/Oracle/Core.hs#L308)?

**Comment by cmika** [comment]
We don't have long term plans to support Oracle so I will not be making the change to Oracle @exodrifter

**Pull request merged**